### PR TITLE
Cloning to workspaces EA

### PR DESCRIPTION
**Pull request by lydia-xie**
# Pull Request/Issue Resolution

Closes #**BD-2270**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+![][1]
+
+When you clone a campaign to a workspace, the following fields are cloned:
+- Campaign name and description
+- Variants
+- Email body
+- Subject and preheader
+
+Note that Content Blocks will not be cloned. However, a Content Block can be referenced in the destination workspace if a block with the same name exists. Alternatively, you can create the Content Block in the destination workspace to avoid any errors when launching a campaign.
```
Comment (willng-braze) [inline _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md:28]: Content is ONE example of liquid references in bodies that are copied over but potentially references another workspace specific object.  There are many others like preference centers, catalogs, custom attributes, etc.

File: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+{% alert important %}
+Cloning campaigns to workspaces is currently in early access. Contact your Braze account manager if you're interested in participating in this early access.
+{% endalert %}
+
+To clone to a workspace, select the <i class="fas fa-cog"></i> gear icon next to the selected campaign, and click **Clone to Workspace**. After cloning to a workspace, we recommend reviewing and testing your campaign to ensure that all fields are working properly.
+
+![][1]
+
+When you clone a campaign to a workspace, the following fields are cloned:
```
Comment (willng-braze) [inline _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md:22]: These fields are specific to EMAIL campaigns.  Other things that are copied are like schedules, conversion behaviors, etc.

File: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+> You can clone campaigns to a different workspace to keep and build off your successful strategies. 
+
+{% alert important %}
+Cloning campaigns to workspaces is currently in early access. Contact your Braze account manager if you're interested in participating in this early access.
+{% endalert %}
+
+To clone to a workspace, select the <i class="fas fa-cog"></i> gear icon next to the selected campaign, and click **Clone to Workspace**. After cloning to a workspace, we recommend reviewing and testing your campaign to ensure that all fields are working properly.
+
+![][1]
```
Comment (willng-braze) [inline _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md:20]: Probably worthwhile to call some big ticket items that aren't copied.  Things like trigger actions and audience filters.

File: _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+# Cloning campaigns to workspaces
+
+> You can clone campaigns to a different workspace to keep and build off your successful strategies. 
+
+{% alert important %}
+Cloning campaigns to workspaces is currently in early access. Contact your Braze account manager if you're interested in participating in this early access.
+{% endalert %}
+
+To clone to a workspace, select the <i class="fas fa-cog"></i> gear icon next to the selected campaign, and click **Clone to Workspace**. After cloning to a workspace, we recommend reviewing and testing your campaign to ensure that all fields are working properly.
```
Comment (willng-braze) [inline _docs/_user_guide/engagement_tools/campaigns/managing_campaigns/cloning_workspaces.md:18]: Might be worthwhile to call out the philosophy as well.   Reword as you see fit.

This approach allows user to get a "jumpstart" on their message composition by starting w/ a copy of another object in a different workspace.  This flow will copy over most bodies and liquid fields but cannot copy over references of objects in the appgroup (for example segments).  we recommend reviewing and testing your campaign to ensure that all fields are working properly.

**Review by bre-fitzgerald — APPROVED** [review]

**Pull request merged**